src/components/app/ProfileSharing.js

```diff
@@ -171,23 +187,40 @@ class ProfileSharingCompositeButton extends PureComponent<
     });
   }
 
-  _attemptToShare() {
-    if (this.state.state !== 'local' && this.state.state !== 'error') {
+  _attemptToShare(isSecondShare: boolean = false) {
```

Comment:
please add a comment on top of this method explaining a bit what `isSecondShare` is and how it changes the behavior of the function